Demo/Demo/Fullscreen/TransactionView/DemoViewModels/PaymentCancelledDemoViewModel.swift

```diff
@@ -23,38 +23,46 @@ extension TransactionDemoViewDefaultData {
         steps: [
             TransactionStepModel(
                 state: .completed,
-                title: "Annonsen er lagt ut",
-                primaryButton: TransactionStepActionButtonModel(
-                    text: "Se annonsen",
-                    style: "flat",
-                    action: "see_ad",
-                    fallbackUrl: "www.finn.no/171529672")),
+                main: TransactionStepContentModel(
+                    title: "Annonsen er lagt ut",
+                    primaryButton: TransactionStepContentActionButtonModel(
+                        text: "Se annonsen",
+                        style: "flat",
+                        action: "see_ad",
+                        fallbackUrl: "/171529672"))),
 
-            TransactionStepModel(
+             TransactionStepModel(
                 state: .completed,
-                title: "Kontrakt",
-                body: NSAttributedString(string: "Begge har signert kontrakten."),
-                primaryButton: TransactionStepActionButtonModel(
-                    text: "Gå til kontrakt",
-                    style: "flat",
-                    url: "https://www.google.com/search?q=contract+signed"
-                )),
+                main: TransactionStepContentModel(
+                    title: "Kontrakt",
+                    body: NSAttributedString(string: "Begge har signert kontrakten."),
+                    primaryButton: TransactionStepContentActionButtonModel(
+                        text: "Gå til kontrakt",
+                        style: "flat",
+                        fallbackUrl: "https://www.google.com/search?q=contract+signed"))),
 
-            TransactionStepModel(
-                state: .completed,
-                title: "Betaling",
-                body: NSAttributedString(string: "Kjøper betalte 1. februar 2020."),
-                detail: "Utbetalingen starter først når begge har bekreftet at overleveringen har skjedd."),
+             TransactionStepModel(
```

Comment:
Example of the difference between `body` and `nativeBody` (along with `nativeButton`)